doc/internal/man3/OPTIONS.pod

```diff
@@ -4,7 +4,8 @@
 
 OPTIONS, OPT_PAIR, OPT_COMMON, OPT_ERR, OPT_EOF, OPT_HELP,
 opt_init, opt_progname, opt_appname, opt_getprog, opt_help,
-opt_begin, opt_next, opt_flag, opt_arg, opt_unknown, opt_cipher, opt_md,
+opt_begin, opt_next, opt_flag, opt_arg, opt_unknown, opt_cipher_basic,
```

Comment:
You did not rename the opt_cipher() to opt_cipher_basic() so it should not be renamed here either.

Reply:
I called it that originally.
Fix coming.